packages/core/core-flows/src/order/utils/order-validation.ts

```diff
@@ -53,6 +53,15 @@ export function throwIfIsCancelled(
   }
 }
 
+export function throwIfPresent(obj: unknown, type: string) {
```

Comment:
question: Can't we have multiple order changes present at the same time? But just not allowed in a few select states, e.g. an open but not requested return.

Reply:
Can we have multiple active order changes? cc @carlos-r-l-rodrigues

Reply:
No, we can only have 1 active order change at time.
They are considered active when the status is "pending" or "requested" which means that the client must accept or decline that order change.

Reply:
I still ended up with a duplicate change order here in this workflow & caused all sorts of ruckus, thats why I added this validation.

Reply:
We'll have to investigate this one more. the root cause doesn't seem to be this validation, as the module is handling this.

Reply:
removed this, can come back to this later! 🤞🏻